cere: upgrade and expand telepad network

warriorstar-orion opened the pull request:
## What Does This PR Do
This PR upgrades the quantum pad in two ways: spawning them with efficiency equal to T4 components, and giving their areas important APCs. It then expands the network, adding stops on the Medbay and Service asteroids, and connecting them to the corners on the opposite sides of the station.

Note that the T4 components don't actually exist in the pads, to prevent them being used for R&D or in other machines. Instead, the pads' efficiency is hard-coded.

## Why It's Good For The Game
Starting the pads upgraded makes sense considering their importance in navigating the station, and increasing their APC cell charge by default ensures they're more reliable as the station goes to shit. Adding more stops in the network makes it much, much easier for people to get around.
## Images of changes

Demo video


https://github.com/ParadiseSS13/Paradise/assets/59303604/4b25fefe-d6f2-4402-9769-0c6a858fde04



### Screenshots

<details><summary>SDMM</summary>

![2024_04_11__19_46_43__paradise dme  cerestation dmm  - StrongDMM](https://github.com/ParadiseSS13/Paradise/assets/59303604/c965d5db-8d40-44b9-bb6a-59eef88217e4)
![2024_04_11__19_46_54__paradise dme  cerestation dmm  - StrongDMM](https://github.com/ParadiseSS13/Paradise/assets/59303604/7e114840-ac04-480d-916a-27e40b19c8bb)
![2024_04_11__19_47_00__paradise dme  cerestation dmm  - StrongDMM](https://github.com/ParadiseSS13/Paradise/assets/59303604/06bb1dc9-b410-4bfe-ae03-0d650c021a46)
![2024_04_11__19_47_05__paradise dme  cerestation dmm  - StrongDMM](https://github.com/ParadiseSS13/Paradise/assets/59303604/1675a4cb-07be-44e3-988d-bb79edc7c43f)
![2024_04_11__19_47_12__paradise dme  cerestation dmm  - StrongDMM](https://github.com/ParadiseSS13/Paradise/assets/59303604/4df5a0e1-e97e-4187-b8e8-edfc5bf42926)
![2024_04_11__20_25_18__paradise dme  cerestation dmm  - StrongDMM](https://github.com/ParadiseSS13/Paradise/assets/59303604/1f5e4bd9-d46c-4316-987e-16e18dc9a32f)


</details>

## Testing
Spawned in, teleported around.


## Changelog
:cl:
add: Farragus: the quantum pad network has been upgraded and expanded. The upgraded parts cannot be taken out of the telepads.
fix: The description for quadratic capacitors was missing a word, which has been fixed.
/:cl:

meow20 commented:
Finally first use of the service direction tag lol

spacegeese commented:
thank you for making fartgas more tolerable

Henri215 commented on code/game/machinery/quantum_pad.dm:
```diff
@@ -45,6 +65,12 @@
 	linked_pad = locate(preset_target)
 
+/obj/machinery/quantumpad/cere/PopulateParts()
+	// No parts in Cere telepads, just hardcode the effiencies
```
```suggestion
	// No parts in Cere telepads, just hardcode the efficiencies
```

DGamerL commented on code/game/objects/items/weapons/stock_parts.dm:
```diff
@@ -220,7 +220,7 @@
 /obj/item/stock_parts/capacitor/quadratic
 	name = "quadratic capacitor"
-	desc = "An capacity capacitor used in the construction of a variety of devices."
+	desc = "An ultra-high capacity capacitor used in the construction of a variety of devices."
```
... how did nobody notice this before

1080pCat reviewed (commented):
Why have you put a vender in the kitchen?
![image](https://github.com/ParadiseSS13/Paradise/assets/96908085/41d3378d-ec2b-4061-9f97-7676cc37f3f5)

Im not the biggest fan of how cramped this tunnel is. there is alot of free _space_ to the right of the area, and the area designation a little odd in the two pointed parts. the lower one could be swapped for the maints area, and the top one could be the unexplored
![image](https://github.com/ParadiseSS13/Paradise/assets/96908085/ceb171d8-79d1-4c88-ac39-72c4696e3cfe)

You've got a stray disposal pipe here
![image](https://github.com/ParadiseSS13/Paradise/assets/96908085/b7f1a1c8-e6e6-4343-b1e6-9cf5cf599aa0)

Stray decal here
![image](https://github.com/ParadiseSS13/Paradise/assets/96908085/fa3b177c-2197-4776-989b-5918c64994f5)

Supply pipes aren't connected to the vent pump
![image](https://github.com/ParadiseSS13/Paradise/assets/96908085/c721facf-1c25-4b4a-b1e6-1668689ad852)

Stray disposal pipe here
![image](https://github.com/ParadiseSS13/Paradise/assets/96908085/2a7e8651-c17a-4487-bce3-859b198e1428)

Light on the shutter
![image](https://github.com/ParadiseSS13/Paradise/assets/96908085/dadbcb17-9127-4fba-8a34-351649eceec6)

Theres no pipes connected to the other scrubbers/vents here
![image](https://github.com/ParadiseSS13/Paradise/assets/96908085/d048e0f5-319c-4520-8d2c-5b8bad3af664)

You've added another display here, but seemingly forgot to remove the original one
![image](https://github.com/ParadiseSS13/Paradise/assets/96908085/0610db28-b24f-4712-aa81-b83fbb7cb00c)

Theres also one here, probably dont need 2 in the same hall in view of eachother
![image](https://github.com/ParadiseSS13/Paradise/assets/96908085/fdae2eec-0e37-4b2d-9356-e484003e1c7e)

On the displays, they're not bumped. but it seems to be like that on all of farragus? so D:

Geology fans love this window.
![image](https://github.com/ParadiseSS13/Paradise/assets/96908085/d5e3170a-8c88-4ef6-bd7d-f4d00e41c02f)

Why have you abbreviated service to `SRUC`? `SERV` would do better.
![image](https://github.com/ParadiseSS13/Paradise/assets/96908085/acfc5418-0336-4771-98f6-640b90768e8b)

This door would be better as a `/public/glass` (the white ones) rather than the grey one
![image](https://github.com/ParadiseSS13/Paradise/assets/96908085/edd4bca8-9632-48cc-9177-e6f2da7eb237)

Could you repipe the pipes to be like this?
![image](https://github.com/ParadiseSS13/Paradise/assets/96908085/43b5d35e-5eb4-4a77-837c-b609d77934b2)

As a final request, could you add fire locks to all of the doors? there are fire alarms, but no.. locks? Im pretty sure you didn't forget them, but since you're redoing them anyways.

warriorstar-orion commented:
> Why have you put a vender in the kitchen? ![image](https://private-user-images.githubusercontent.com/96908085/323006239-41d3378d-ec2b-4061-9f97-7676cc37f3f5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMzMDcxMTcsIm5iZiI6MTcxMzMwNjgxNywicGF0aCI6Ii85NjkwODA4NS8zMjMwMDYyMzktNDFkMzM3OGQtZWMyYi00MDYxLTlmOTctNzY3NmNjMzdmM2Y1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE2VDIyMzMzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZlZmRlNWY4ZWI2MjYyNDY2MzdiMjQ4MTk1OWVhOGE2ZjJkNjFiMWRhODA0NDhiZDc2NTZkMzI2ODVkNzkyMDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.EfNlmu5D9jOFIakdoux_mk1BSQIg8vA-voEv6iXHRLY)

Originally the pads opened straight into the bar, this was an artifact of that. Removed.

> 
> Im not the biggest fan of how cramped this tunnel is. there is alot of free _space_ to the right of the area, and the area designation a little odd in the two pointed parts. the lower one could be swapped for the maints area, and the top one could be the unexplored ![image](https://private-user-images.githubusercontent.com/96908085/323006762-ceb171d8-79d1-4c88-ac39-72c4696e3cfe.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMzMDcxMTcsIm5iZiI6MTcxMzMwNjgxNywicGF0aCI6Ii85NjkwODA4NS8zMjMwMDY3NjItY2ViMTcxZDgtNzlkMS00Yzg4LWFjMzktNzJjNDY5NmUzY2ZlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE2VDIyMzMzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE5ZDQzOGY2MDllOTgyZjk5MzY3MThmN2VjMmZmZTQ0ODBkMDk5NGI5YjU1N2VlY2NiNzU1OTFkNDBkYTU3ZmImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.zpnZgpp1oP7c91eXHq3p_rAAfRl13eH_l7CNNWrABVY)

I've expanded this a bit and fixed the two areas you mentioned, PTAL. Regarding whether rock walls in maints count as the maint area or unexplored mining, Cere does a little bit of both. Port maintenance includes the rock walls in the maint area, but med maints only includes constructed walls. I went with the latter but there should probably be a standard around this.

> 
> You've got a stray disposal pipe here ![image](https://private-user-images.githubusercontent.com/96908085/323007371-b7f1a1c8-e6e6-4343-b1e6-9cf5cf599aa0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMzMDcxMTcsIm5iZiI6MTcxMzMwNjgxNywicGF0aCI6Ii85NjkwODA4NS8zMjMwMDczNzEtYjdmMWExYzgtZTZlNi00MzQzLWIxZTYtOWNmNWNmNTk5YWEwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE2VDIyMzMzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY2NTg2ODcwNjhlZGFjZTFhZjE3N2ZlMjkyOWJjOTQ1MzNmMTUyZDJmNWIxZjJkOTg2MWM2ZmRlMTFmNjRmYzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.A4DJwo0I7u9mJMTlFjKlsMk9JND3dVc8jY_14Bbd8OM)
> 

Fixed.

> Stray decal here ![image](https://private-user-images.githubusercontent.com/96908085/323007593-fa3b177c-2197-4776-989b-5918c64994f5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMzMDcxMTcsIm5iZiI6MTcxMzMwNjgxNywicGF0aCI6Ii85NjkwODA4NS8zMjMwMDc1OTMtZmEzYjE3N2MtMjE5Ny00Nzc2LTk4OWItNTkxOGM2NDk5NGY1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE2VDIyMzMzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTliNjQxMjY2Y2M1MmJjYjIxYWE5NTE5MjI5NDBjODQwZGUyYjdjMGNlZjljOTMzN2ZkOTJlMDQ1NDc1NGVhYTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.luLcQ8SOLqSj3wQTLwXv7Ar1khr0mMqTxcBZSLHKtrk)

Fixed.


> 
> Supply pipes aren't connected to the vent pump ![image](https://private-user-images.githubusercontent.com/96908085/323008242-c721facf-1c25-4b4a-b1e6-1668689ad852.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMzMDcxMTcsIm5iZiI6MTcxMzMwNjgxNywicGF0aCI6Ii85NjkwODA4NS8zMjMwMDgyNDItYzcyMWZhY2YtMWMyNS00YjRhLWIxZTYtMTY2ODY4OWFkODUyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE2VDIyMzMzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdhYjYxMGRmODk5NDRmYWNhNmZiNjE5YTY3OTUyOWQxNDJlNTRjYWRkYjBlYzUwYjdhMzI1MTY2NjQ2NDY1NDEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.2Me-k7vGn3H7nQVJxhkmh-vUAEo97J-vHI9YG6X73d4)
> 

Fixed.

> Stray disposal pipe here ![image](https://private-user-images.githubusercontent.com/96908085/323008321-2a7e8651-c17a-4487-bce3-859b198e1428.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMzMDcxMTcsIm5iZiI6MTcxMzMwNjgxNywicGF0aCI6Ii85NjkwODA4NS8zMjMwMDgzMjEtMmE3ZTg2NTEtYzE3YS00NDg3LWJjZTMtODU5YjE5OGUxNDI4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE2VDIyMzMzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWEwZjIzNmZlNzE3YTRlNzA1YjQzZmRlYjc1MTVjMjQ1YTc1ZWIyMmM3NTg3OGRhYWZkNzg5ZGMwZDIyNWU5MjEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.8JBTdW6bsM6_b0hLPTjaHVj3J3kCyz4bXx1QZpFRWRU)
> 

Fixed.

> Light on the shutter ![image](https://private-user-images.githubusercontent.com/96908085/323008468-dadbcb17-9127-4fba-8a34-351649eceec6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMzMDcxMTcsIm5iZiI6MTcxMzMwNjgxNywicGF0aCI6Ii85NjkwODA4NS8zMjMwMDg0NjgtZGFkYmNiMTctOTEyNy00ZmJhLThhMzQtMzUxNjQ5ZWNlZWM2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE2VDIyMzMzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE3MDI0YTNlYWNjMGQ5NWFlOTgwNzg3Y2JhYzBkZDNmZWE4NzhhNWFlNmZjOThhYTBjNzRhN2FiYmU1ODc1NTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.c9qsn_6hpTA7XtwVr7mhE72Hhq6V_EdXXHD4UqKApPc)

Fixed. 


> Theres no pipes connected to the other scrubbers/vents here ![image](https://private-user-images.githubusercontent.com/96908085/323008689-d048e0f5-319c-4520-8d2c-5b8bad3af664.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMzMDcxMTcsIm5iZiI6MTcxMzMwNjgxNywicGF0aCI6Ii85NjkwODA4NS8zMjMwMDg2ODktZDA0OGUwZjUtMzE5Yy00NTIwLThkMmMtNWI4YmFkM2FmNjY0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE2VDIyMzMzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM5MmFkZTk4Y2MxZjJjMDQ2MzA2NDgzNGU5MzkyZmU5YjBjOTRmNjQ0MjQ2YTlmNmZiZjk2ZDc5NThiMmJlNGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.31qffvV5f4TPyATkatk6dCT_AHZZNGtplZa9FtmEIjI)
> 
> You've added another display here, but seemingly forgot to remove the original one ![image](https://private-user-images.githubusercontent.com/96908085/323008918-0610db28-b24f-4712-aa81-b83fbb7cb00c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMzMDcxMTcsIm5iZiI6MTcxMzMwNjgxNywicGF0aCI6Ii85NjkwODA4NS8zMjMwMDg5MTgtMDYxMGRiMjgtYjI0Zi00NzEyLWFhODEtYjgzZmJiN2NiMDBjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE2VDIyMzMzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM1Y2EzZDdjY2RhMGM4MzRkZTA2OTIxMWU5OTI1YzEwYmVhNmU1ZmQ2MWM1MzhkMWMwMzYxN2RlYThiM2IzMmImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.3x9yVXwdZi9o3VE0Qk29sGPA_dwgWWPw48whobRlcRA)
> 

Fixed.

> Theres also one here, probably dont need 2 in the same hall in view of eachother ![image](https://private-user-images.githubusercontent.com/96908085/323009020-fdae2eec-0e37-4b2d-9356-e484003e1c7e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMzMDcxMTcsIm5iZiI6MTcxMzMwNjgxNywicGF0aCI6Ii85NjkwODA4NS8zMjMwMDkwMjAtZmRhZTJlZWMtMGUzNy00YjJkLTkzNTYtZTQ4NDAwM2UxYzdlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE2VDIyMzMzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU0ODhkNjg4MjZiNTM4MTk5NmJmNmM5MDIyY2U3M2UzOTYxZDVhNmFhNjA0YmRjYjkxNGE0MmUwYjBhMTQ5NDAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.EJrbvQpGRhPqGMJ4jv00wsBJiGLdXqxnQDJ_GRCAMKk)
> 

Fixed.

> On the displays, they're not bumped. but it seems to be like that on all of farragus? so D:
 
I'll probably fix all these when I make directional helpers for them.

> Geology fans love this window. ![image](https://private-user-images.githubusercontent.com/96908085/323009762-d5e3170a-8c88-4ef6-bd7d-f4d00e41c02f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMzMDcxMTcsIm5iZiI6MTcxMzMwNjgxNywicGF0aCI6Ii85NjkwODA4NS8zMjMwMDk3NjItZDVlMzE3MGEtOGM4OC00ZWY2LWJkN2QtZjRkMDBlNDFjMDJmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE2VDIyMzMzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI3NWZjZWNkNzY1Mjc3YTY1MjM1MDJkM2Q4ZTY0N2E4YjhkODM2YWVmOGQ0ZTEwYmRkOWNhOTY1YjlhZmM5MjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.hdCk78Ei16LHGQs4j4rD4dhPCqNQQW_nApauCxQ_NFs)

Fixed.

> Why have you abbreviated service to `SRUC`? `SERV` would do better. ![image](https://private-user-images.githubusercontent.com/96908085/323010836-acfc5418-0336-4771-98f6-640b90768e8b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMzMDcxMTcsIm5iZiI6MTcxMzMwNjgxNywicGF0aCI6Ii85NjkwODA4NS8zMjMwMTA4MzYtYWNmYzU0MTgtMDMzNi00NzcxLTk4ZjYtNjQwYjkwNzY4ZThiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE2VDIyMzMzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBhYzdjMzI0NzBhMWRiM2UxOWIwNWY5YjNlNmQxYWY0ZjRkMjg0ZmU0OWEwZjc2NDhjMGMyZWQ0MTYzNGI4OGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.6t_pm6KwbOqT5YKHUcp7IZIgxru8bQglaGxtJtxip90)

That's the same V used in the sign for "EVAC", and was meant to be an abbreviation of SeRViCe. I've changed it to SERV and darkened the colors a tiny bit so that it looks more like a V and less like a U.

> This door would be better as a `/public/glass` (the white ones) rather than the grey one ![image](https://private-user-images.githubusercontent.com/96908085/323011852-edd4bca8-9632-48cc-9177-e6f2da7eb237.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMzMDcxMTcsIm5iZiI6MTcxMzMwNjgxNywicGF0aCI6Ii85NjkwODA4NS8zMjMwMTE4NTItZWRkNGJjYTgtOTYzMi00OGNjLTkxNzctZTZmMmRhN2ViMjM3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE2VDIyMzMzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQxODBiYWMwZjY2ZDg5NWIyNzJiZmUwMGFiOWFhNWVhOTI3NGY2MDY4Y2M5ZmE0MGJiNDA5NTUyNTIxYmNhYjUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.MqdzCU9Skae-Zyi8s1qjlFDLM9x7WV1ev0lqUXtxewU)

Fixed.

> Could you repipe the pipes to be like this? ![image](https://private-user-images.githubusercontent.com/96908085/323012438-43b5d35e-5eb4-4a77-837c-b609d77934b2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMzMDcxMTcsIm5iZiI6MTcxMzMwNjgxNywicGF0aCI6Ii85NjkwODA4NS8zMjMwMTI0MzgtNDNiNWQzNWUtNWViNC00YTc3LTgzN2MtYjYwOWQ3NzkzNGIyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE2VDIyMzMzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI4OThlMDY5ODlmNzZkMTUxNjIxNmFjZGZhMWUzMzZjODA1OThkNzkxZjAxMzFkODEyMTllYmRlYzVlYzRhMmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.xtctymXGCLPe1uLcncmiT8VAeTQy_vddMDwulzDfObo)
> 

Fixed.

> As a final request, could you add fire locks to all of the doors? there are fire alarms, but no.. locks? Im pretty sure you didn't forget them, but since you're redoing them anyways.

Fixed.
